doc/restore.md

```diff
@@ -0,0 +1,21 @@
+# Remove the Windows Package Manager preview
+
+Sometimes users trying the preview of the Windows Package Manager would like to restore their machine to a version that did not include the preview.  The following steps will restore your machine.
+
+1) Remove your machine from Windows Insider Flights.
+The Windows Package Manager is included in the Windows Insider Flights.  If you are installing a Windows Insider build, you will keep getting updated.
+2) Remove yourself from the Windows Package Manager flights.  
+If you requested to be added to the Windows Package Manager flights, then you will specifically need to ask to be removed.  You can ask to be removed by filling in the following form:
+
+Note: Make sure you use the correct email address.  It must be the address used to sign up for the Insider build.  If your email address is not the same address that you are using to log into the Windows Store, then removing it will have no impact.
+
+Please allow for 1 week, to be sure the form has been processed, though most likely it is processed much sooner.
+
+3) Install an older build of the [Desktop App Installer](https://www.microsoft.com/en-us/p/app-installer/9nblggh4nns1).  We have placed an earlier released version [here](https://www.github.com/micrsoft/winget-cli/releases).
```

Comment:
I think we don't want to use "here" for links? I read somewhere that it makes it harder to parse what the link refers to, both for people and tools like screen readers.

Maybe something like `Here is an [earlier released version for download](...).`

Reply:
I'd probably write `There is...` or `An [earlier version(...) is available for download.`

And yes, avoid linking `here` it's screen reader hostile.